CONTRIBUTING.md

```diff
@@ -16,3 +16,32 @@ rover is an open source project developed by [Apollo GraphQL](https://apollograp
   - Your PR must not have any errors from `cargo clippy`.
 - One approval from a member of the Apollo tooling team.
   - The team consists of @jbaxleyiii, @ashleygwilliams, @JakeDawkins, and @EverlastingBugstopper.
+
+## Adding new commands/subcommands
+
+When adding a new command or a subcommand, you should make sure that the field in the new command struct is titled `command`. This allows our automated anonymous usage collection to work properly.
+
+Should you add arguments that are _not_ subommands, the arguments you specify will automatically be reported to Apollo's Telemetry backend, unless you specify that an argument should be ignored with `#[serde(skip_serializing)]`. You should be careful to add the `skip_serializing` attribute to any fields that may contain data that may not be anonymous. This can be anything from a profile name to a path on the user's computer that could include their name.
```

Comment:
```suggestion
Should you add arguments that are _not_ subcommands, the arguments you specify will automatically be reported to Apollo's Telemetry backend, unless you specify that an argument should be ignored with `#[serde(skip_serializing)]`. You should be careful to add the `skip_serializing` attribute to any fields that may contain data that may not be anonymous. This can be anything from a profile name to a path on the user's computer that could include their name.
```